plugins/callback_declarations.h

```diff
@@ -51,7 +51,7 @@
 	PCL void OnPlayerWantReservedSlot(netadr_t* from, char* pbguid, char* userinfo, int authstate, qboolean *isallowed);
 	PCL void OnModuleLoaded(client_t* client, char* fullpath, long checksum);
 	PCL void OnScreenshotArrived(client_t* client, const char* path);
-	
+	PCL void OnDemoArrived(client_t* client,char *demoname);
```

Comment:
Not sure if it's a good idea to let the plugin API users modify the demo name when the demo file has already been written to disk. It would give the assumption that the filename can be changed from within the callback, which is not the case.
```suggestion
	PCL void OnDemoArrived(client_t* client, const char *demoname);
```